scipy/signal/filter_design.py

```diff
@@ -252,8 +254,8 @@ def freqs_zpk(z, p, k, worN=None):
     return w, h
 
 
-def freqz(b, a=1, worN=None, whole=False, plot=None):
-    """
+def freqz(b, a=1, worN=512, whole=False, plot=None, axis=0):
```

Comment:
> I think worN=None is preferable. Suppose I write a function that calls freqz, and I want my function to have a worN argument that is passed on to freqz. If freqz gives the default in the signature, I have to copy that default to my function's signature.

But the same argument could be made about *any* argument that has a non-None default, right?